org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/entities/EntitiesDefinitionExtensionsTest.java

```diff
@@ -75,6 +75,19 @@ public void insideText() throws BadLocationException {
 		testDefinitionFor(xml, "test.xml", ll("test.xml", r(5, 7, 5, 12), r(2, 11, 2, 16)));
 	}
 
+	@Test
+	public void underscoreEntityName() throws BadLocationException {
+		String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<!DOCTYPE article [\r\n" + //

```

Comment:
shouldn't it be `DOCTYPE  root` to be more consistent?

Reply:
fixed